client/my-sites/stats/stats-date-control/stats-date-control-picker-date.tsx

```diff
@@ -16,8 +16,14 @@ const DateControlPickerDate = ( {
 	return (
 		<div className="date-control-picker-date">
 			<div className="stats-date-control-picker-dates__inputs">
-				<TextControl value={ startDate } onChange={ onStartChange } />
-				<TextControl value={ endDate } onChange={ onEndChange } />
+				<div className="input-group">
+					<label htmlFor="startDate">From</label>
+					<TextControl id="startDate" value={ startDate } onChange={ onStartChange } />
+				</div>
+				<div className="input-group">
```

Comment:
Let's make `input-group` a more specific name to avoid CSS leaks.

Reply:
Fixed in 0fac63def3